src/unicode.cpp

```diff
@@ -44,7 +46,27 @@ constexpr bool to_lower_ascii(char* input, size_t length) noexcept {
   }
   return non_ascii == 0;
 }
-#if ADA_NEON
+#endif
+#if ADA_SVE
+ada_really_inline bool has_tabs_or_newline(
+    std::string_view user_input) noexcept {
+  const svuint8_t mask1 = svdup_n_u8('\r');
+  const svuint8_t mask2 = svdup_n_u8('\n');
+  const svuint8_t mask3 = svdup_n_u8('\t');
+  svbool_t running = svdup_n_b8(false);
+  const size_t lanes = svcntb();
+  for (size_t i = 0; i < user_input.size(); i += lanes) {
```

Comment:
what happens if lanes is bigger than user_input.size() initially? wouldn't there be a overflow?

Reply:
If `lanes > user_input.size()` , e.g. `lanes = 16` and `user_input.size() = 10` , then the loop runs only once and the `mask` becomes `1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0` . No overflow happens.